app/gateway/changelog.md

```diff
@@ -3,6 +3,43 @@ title: Kong Gateway Changelog
 no_version: true
 ---
 
+<!-- vale off -->
+
+## 2.8.1.2
+**Release Date** 2022/06/28
+
+### Fixes
+
+#### Enterprise
+
+* Fixed an issue in hybrid mode where, if a service was set to `enabled: false` and that service had a route with an enabled plugin, any new data planes would receive empty configuration.
+* Fixed a timer leak that occurred when `worker_consistency` was set to `eventual` in `kong.conf`.
+The issue caused timers to be exhausted and failed to start any other timers used by Kong Gateway, showing the error `too many pending timers`.
+
+#### Kong Manager and Dev Portal
+
+* Fixed an issue where Kong Manager did not display all Dev Portal developers in the organization.
+* Fixed an issue with displaying developer role assignment in Kong Manager.
+When viewing a role under the Permissions tab in the Dev Portal section, the list of developers wouldn't update when a new developer was added.
+Kong Manager was constructing the wrong URL when retrieving Dev Portal assignees.
```

Comment:
```suggestion
Fixed an issue that caused Kong Manager to construct the wrong URL when it retrieved Dev Portal assignees.
```
Just reworked this to match some of the other "Fixed an issue..." type language and to change the verb tenses.

Reply:
This isn't a new entry but an explanation for "Fixed an issue that prevented developer role assignments from displaying in Kong Manager." There was an issue, you wouldn't see devs in the list, and it happened because Kong Manager was constructing the wrong URL. So the user wouldn't see the wrong URL, they would see the symptom, which is the missing developers.

Reply:
That makes sense!